PRESUBMIT.py

```diff
@@ -54,6 +70,36 @@ def CheckPatchFormatted(input_api, output_api):
 
     is_format_required = git_cl_format_code == 2
 
+    if not is_format_required or input_api.PRESUBMIT_FIX:
+        # Use Prettier to format other file types.
+        prettier_types = (
+            '.js',
+            '.ts',
+            '.tsx',
+        )
```

Comment:
I suppose we can add .html, .css, and .scss when we add and configure stylelint

Reply:
Prettier also supports .md & .yml, I think